src/pages/Explore/index.tsx

```diff
@@ -0,0 +1,196 @@
+  }
+`
+const SearchContainer = styled(FiltersContainer)<{ isExplore: boolean }>`
+  ${({ isExplore }) => !isExplore && 'margin-left: 8px;'}
+  width: 100%;
+
+  @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
+    ${({ isExplore }) => !isExplore && 'order: 1; margin: 0px;'}
+    ${({ isExplore }) => isExplore && 'justify-content: flex-end;'}
+  }
+`
+const NavWrapper = styled.div<{ isExplore: boolean }>`
+  display: flex;
+  max-width: ${MAX_WIDTH_MEDIA_BREAKPOINT};
+  margin: 0 auto;
+  margin-bottom: ${({ isExplore }) => (isExplore ? '16px' : '20px')};
+  color: ${({ theme }) => theme.neutral3};
+  flex-direction: row;
+
+  @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
+    flex-direction: column;
+    gap: ${({ isExplore }) => (isExplore ? '16px' : '8px')};
+  }
+`
+
+const Pages: Array<TabbedNav> = [
+  {
+    title: <Trans>Tokens</Trans>,
+    key: 'tokens',
+    component: TokenTable,
+    loggingElementName: 'explore-tokens-tab', // todo: add to InterfaceElementName @uniswap/analytics-events
+  },
+  {
+    title: <Trans>Pools</Trans>,
+    key: 'pools',
+    component: TokenTable,
+    loggingElementName: 'explore-pools-tab',
+  },
+  {
+    title: <Trans>Transactions</Trans>,
+    key: 'transactions',
+    component: TokenTable,
+    loggingElementName: 'explore-transactions-tab',
+  },
+]
+
+const Explore = () => {
+  const resetFilterString = useResetAtom(filterStringAtom)
+  const location = useLocation()
+  const [currentTab, setCurrentTab] = useState(0)
+  const isExplore = useInfoExplorePageEnabled()
+
+  useEffect(() => {
+    resetFilterString()
+  }, [location, resetFilterString])
+
+  const { component: Page, key: currentKey } = Pages[currentTab]
+
+  return (
+    // TODO: add 'explore-page' to InterfacePageName in @uniswap/analytics-events
+    <Trace page={isExplore ? 'explore-page' : InterfacePageName.TOKENS_PAGE} shouldLogImpression>
+      <ExploreContainer>
+        {/* TODO: add graphs to explore page */}
+        {isExplore ? (
+          <div></div>
+        ) : (
+          <TitleContainer>
+            <MouseoverTooltip
+              text={<Trans>This table contains the top tokens by Uniswap volume, sorted based on your input.</Trans>}
+              placement="bottom"
+            >
+              <ThemedText.LargeHeader>
+                <Trans>Top tokens on Uniswap</Trans>
+              </ThemedText.LargeHeader>
+            </MouseoverTooltip>
+          </TitleContainer>
+        )}
+        <NavWrapper isExplore={isExplore}>
+          {isExplore && (
+            <Nav data-testid="explore-navbar">
+              {Pages.map(({ title, loggingElementName, key }, index) => {
+                const handleNavItemClick = () => {
+                  setCurrentTab(index)
+                }
+                return (
+                  <TraceEvent
+                    events={[BrowserEvent.onClick]}
+                    name={SharedEventName.NAVBAR_CLICKED}
```

Comment:
this is not the correct event `NAVBAR_CLICKED` is specifically referring to the app wide NavBar at the top of the page. This would confuse our analytics and should be removed.

Reply:
hmm - SharedEventName is an enum for 'General event names that can be reused across products' - and MiniPortfolio also currently uses this name for its tabbed navbar. we disambiguate with the element name, which is more specific